contrib/utilities/wrapcomments.py

```diff
@@ -34,9 +46,11 @@
 # (inadvertently) split across two lines.
 wrapper = textwrap.TextWrapper(break_on_hyphens=False)
 
-# take an array of lines and wrap them to 78 columns and let each line start
-# with @p startwith
+# Take an array of lines and return them with @p startwith prepended
 def wrap_block(lines, startwith):
+    return [startwith+line for line in lines]
+
+    # We used to rewrap the text to 78 columns:
     longline = " ".join(lines)
     wrapper.initial_indent = startwith
     wrapper.subsequent_indent = startwith
```

Comment:
I'm confused here. Doesn't the `return` statement pre-empt all of the following indented lines?